frontend/models/product-list/index.ts

```diff
@@ -47,16 +47,17 @@ export type {
 export async function findProductList(
    filters: ProductListFiltersInput
 ): Promise<ProductList | null> {
-   const result = await strapi.getProductList({
-      filters,
-   });
+   const deviceTitle = filters.deviceTitle?.eq ?? '';
 
+   const [result, deviceWiki] =
+      await Promise.all([
+         strapi.getProductList({
+            filters,
+         }),
+         fetchDeviceWiki(deviceTitle)
+      ]);
```

Comment:
I like how succinct this is. I would personally avoid calling `fetchDeviceWiki(deviceTitle)` if `deviceTitle === ''`, but the result is the same so I'm fine with it either way.

```suggestion
   const [result, deviceWiki] =
      await Promise.all([
         strapi.getProductList({
            filters,
         }),
         deviceTitle ? fetchDeviceWiki(deviceTitle) : null,
      ]);
```

Reply:
I like handling it in the function so if anyone uses the function in future they don't have to account for their input being `== null`